Assets/Scripts/Controllers/TileSpriteController.cs

```diff
@@ -90,4 +109,15 @@ protected override void OnChanged(Tile tile)
     protected override void OnRemoved(Tile tile)
     {
     }
+
+    private void ChangeTileSprite(GameObject tile_go, string name)
+    {
+        // TODO How to manage if not all of the names are present?
+        tile_go.GetComponent<SpriteRenderer>().sprite = SpriteManager.current.GetSprite("Tile", name);
+    }
+
+    private bool DoesTileSpriteExist(string name)
```

Comment:
Yes, and in fact, the only tile type that does is the Floor, so... Also, this allows for loading it to the correct sprite, without changing the sprite every time.